mscore/plugin/pluginCreator.ui

```diff
@@ -89,7 +89,7 @@
      <x>0</x>
      <y>0</y>
      <width>800</width>
-     <height>21</height>
+     <height>20</height>
```

Comment:
Qt Creator is funny sometimes

Reply:
Yep. Just don't commit that

Reply:
yeah, on command line in *nix I don't mess with trying to omit per-line. Reminds me of some stuff back in 3.4-3.6 days :) At any rate, thanks for the recommendation to alternatively save when editor has changes.